src/main/java/io/confluent/kafkarest/ConsumerManager.java

```diff
@@ -55,9 +56,9 @@
 
   private final KafkaRestConfig config;
   private final Time time;
-  private final String zookeeperConnect;
+  private final String bootstrapServers;
   private final MetadataObserver mdObserver;
-  private final int iteratorTimeoutMs;
+//  private final int iteratorTimeoutMs;
```

Comment:
Please delete any commented code. Better to not check this in and instead let version control track files that have been deleted/added.